src/pages/Items/Product.jsx

```diff
@@ -0,0 +1,368 @@
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+`;
+
+const Inquiry = styled.div`
+  font-size: 18px;
+  color: #111827;
+`;
+
+const InquiryInput = styled.input`
+  width: 100%;
+  padding: 16px 24px;
+  box-sizing: border-box;
+  height: 104px;
+  display: flex;
+  gap: 10px;
+  color: #1f2937;
+  border-radius: 12px;
+  background-color: #f3f4f6;
+  border: none;
+`;
+
+const InquiryListWrapper = styled.div`
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+`;
+
+const BackToListButton = styled.button`
+  width: 240px;
+  border-radius: 40px;
+  display: flex;
+  gap: 2ㅎ0px;
+  padding: 12px 0;
+  background-color: #3692ff;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  cursor: pointer;
+  border: none;
+
+  font-family: Pretendard;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 26px;
+  letter-spacing: 0%;
+  text-align: center;
+  color: #f3f4f6;
+
+  &::after {
+    content: "";
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+    background-image: url("/images/ic_back.png");
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
+`;
+
+const LinkProduct = styled(Link)`
+  text-decoration: none;
+`;
+
+const Product = () => {
+  const [product, setProduct] = useState(null);
+  const [inquiryText, setInquiryText] = useState("");
+  const [comments, setComments] = useState([]);
+  const [likeCount, setLikeCount] = useState(0);
+  const { productId } = useParams();
+
+  useEffect(() => {
+    fetch(`https://panda-market-api.vercel.app/products/${productId}`)
+      .then((res) => res.json())
+      .then((data) => {
+        setProduct(data);
+        setLikeCount(data.favoriteCount || 0);
+        setComments(
+          (data.comments || []).map((c, i) => ({ ...c, id: Date.now() + i }))
+        );
+      })
+      .catch((err) => console.error("데이터 불러오기 실패:", err));
+  }, [productId]);
+
+  const isFormValid = inquiryText.trim().length > 0;
+
+  const handleAddComment = () => {
+    const newComment = {
+      id: Date.now(),
+      image: "/images/ic_profile.png",
+      nickname: "익명사용자",
+      content: inquiryText,
+      updatedAt: new Date().toLocaleString("ko-KR", { hour12: false }),
+    };
+
+    setComments([newComment, ...comments]);
+    setInquiryText("");
+  };
+
+  const handleDeleteComment = (id) => {
+    setComments((prev) => prev.filter((c) => c.id !== id));
+  };
+
+  const handleEditComment = (id, newText) => {
+    setComments((prev) =>
+      prev.map((c) => (c.id === id ? { ...c, content: newText } : c))
+    );
+  };
+
+  const handleLike = () => {
+    setLikeCount((prev) => prev + 1);
+  };
+
+  return (
+    <>
+      <Header />
+      <Container>
+        {product && (
+          <>
+            <ProductWrapper>
+              <ProductImage>
+                {product.images && (
+                  <img src={product.images[0]} alt={product.name} />
+                )}
```

Comment:
💬 여담
product의 기본값을 빈객체로 설정하면 아래처럼 작성이 가능합니다.
지금처럼 product 값이 있을때 UI가 보인다면, 로딩중에는 해당 영역이 다 노출되지 않으니 스켈레톤을 추가해주시는 것처럼 영역을 유지할 수 있게 작업하시면 좋겠습니다.

```suggestion
//  const [product, setProduct] = useState({});
        <>
          <ProductWrapper>
            <ProductImage>
              {product?.images && (
                <img src={product.images[0]} alt={product.name} />
              )}
```